conans/client/generators/cmake_find_package.py

```diff
@@ -19,14 +19,15 @@ class CMakeFindPackageGenerator(Generator):
 
         include(FindPackageHandleStandardArgs)
 
-        conan_message(STATUS "Conan: Using autogenerated Find{name}.cmake")
+        conan_message(STATUS "Conan: Using autogenerated Find{filename}.cmake")
         # Global approach
-        set({name}_FOUND 1)
+        set({filename}_FOUND 1)
+        {set_found_filename_uppercase}
         set({name}_VERSION "{version}")
 
         find_package_handle_standard_args({name} REQUIRED_VARS
                                           {name}_VERSION VERSION_VAR {name}_VERSION)
-        mark_as_advanced({name}_FOUND {name}_VERSION)
+        mark_as_advanced({filename}_FOUND {name}_VERSION)
```

Comment:
Mark both variables as advanced
```suggestion
        mark_as_advanced({filename}_FOUND {filename_upper}_FOUND {name}_VERSION)
```